qucs/qucs/Makefile.am

```diff
@@ -100,15 +104,16 @@ EXTRA_DIST = \
 	bitmaps/qucsrescodes.icns \
 	bitmaps/qucstrans.icns
 
-SUFFIXES = .h .moc.cpp .qrc.cpp
+SUFFIXES = .h .moc.cpp .qrc
 
-.qrc.cpp.cpp:
-qucs.qrc.cpp: qucs.qrc
+.qrc.cpp:
```

Comment:
my understanding is, that `.qrc.cpp:` instructs automake to use the path `.qrc -> .cpp -> .o`, where applicable.

cf https://www.gnu.org/software/automake/manual/html_node/Suffixes.html